src/index.ts

```diff
@@ -9,35 +10,22 @@ const movieSearchBox = document.getElementById(
   'movie-search-box'
 ) as HTMLInputElement;
 const searchList = document.getElementById('search-list') as HTMLInputElement;
+let searchTerm = movieSearchBox.value.trim();
```

Comment:
Does this work? It looks like `searchTerm` is only given a value once on page load, and never again after that. So I would think searches wouldn't update.

Reply:
Yes i moved this to the click event.